site/specs/numbers.json

```diff
@@ -5040,100 +5018,91 @@
+            "description": "<p>Checks the order's creation date against this value. Orders that have a creation date after this date will be included. Format is yyyy-MM-dd</p>",
             "in": "query",
             "required": false,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "2013-10-22"
           },
           {
             "name": "createdDateTo",
-            "description": "<p>Checks the order's creation date against this value. Orders that have a creation date before this date will be included. Format is yyyy-MM-dd</p>\n",
+            "description": "<p>Checks the order's creation date against this value. Orders that have a creation date before this date will be included. Format is yyyy-MM-dd</p>",
             "in": "query",
             "required": false,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "2013-10-25"
           },
           {
             "name": "orderIdFragment",
-            "description": "<p><b class=\"required\">*</b> This search parameter represents the internal Bandwidth Dashboard API ID that has been assigned to the TN Option Order.  This parameter is the first few characters of the internal ID - the entire ID does not need to be specified.</p>\n",
+            "description": "<p><b class=\"required\">*</b> This search parameter represents the internal Bandwidth Dashboard API ID that has been assigned to the TN Option Order.  This parameter is the first few characters of the internal ID - the entire ID does not need to be specified.</p>",
             "in": "query",
             "required": false,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "ed1c0bed-e2"
           },
           {
             "name": "action",
-            "description": "<p>whether or not the results should be restricted to IMPORT or REMOVE orders.\"</p>\n",
+            "description": "<p>whether or not the results should be restricted to IMPORT or REMOVE orders.</p>",
             "in": "query",
             "required": false,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "REMOVE"
           },
           {
             "name": "loaType",
-            "description": "<p>whether or not the results should be restricted to CARRIER or SUBSCRIBER orders.  The Default value is \"ALL\"</p>\n",
+            "description": "<p>whether or not the results should be restricted to CARRIER or SUBSCRIBER orders.  The Default value is \"ALL\"</p>",
```

Comment:
remove `The Default value is \"ALL\"`  and add `"default": "ALL"` to the `schema`.